### PR TITLE
Fix WordPress.Security.ValidatedSanitizedInput.InputNotValidated error

### PR DESCRIPTION
**Pull request by jspellman814**
### Description of the Change

This PR fixes the following error thrown by the VIP Code Analysis Bot:
```
Error( severity 5 ): Detected usage of a possibly undefined superglobal array index: $_SERVER['REQUEST_URI']. Use isset() or empty() to check the index exists before using it (WordPress.Security.ValidatedSanitizedInput.InputNotValidated).
```

If it doesn't make sense for you all to merge the work in this PR, feel free to close. Thank you!

Closes #

### Changelog Entry

> Changed - Existing functionality

Address the issue of a possibly undefined superglobal array index by introducing a concise and effective way to handle potential null values.

### Credits

Props @jspellman814

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Review by ravinderk — APPROVED** [review]

File: inc/classes/class-srm-redirect.php [hunk 1]
```diff
@@ -285,7 +285,7 @@ public function maybe_redirect() {
 		 * @param {string} $request_path Request path. Default `$_SERVER['REQUEST_URI']`.
 		 * @returns {string} Request path.
 		 */
-		$requested_path   = esc_url_raw( apply_filters( 'srm_requested_path', $_SERVER['REQUEST_URI'] ) );
+		$requested_path   = esc_url_raw( apply_filters( 'srm_requested_path', $_SERVER['REQUEST_URI'] ?? '' ) );
```
Comment (ravinderk) [inline inc/classes/class-srm-redirect.php]: @jspellman814, can we also sanitize this value?

**Review by ravinderk — CHANGES_REQUESTED** [review]
@jspellman814 Thank you for your contribution. I added a minor request. Feel free to ask questiona.

File: inc/classes/class-srm-redirect.php (hunk 1 above)
Comment (ravinderk) [reply inc/classes/class-srm-redirect.php]: After this patch, I am seeing the following warning - 

image

Comment (jspellman814) [reply inc/classes/class-srm-redirect.php]: @ravinderk I pushed a fix for this in [07db92f](https://github.com/10up/safe-redirect-manager/pull/356/commits/07db92fa6406daf15612e84b3a4817cb725bbad7).

**Review by ravinderk — APPROVED** [review]

**Pull request merged**